core/src/mindustry/world/blocks/liquid/LiquidRouter.java

```diff
@@ -2,6 +2,8 @@
 
 import mindustry.gen.*;
 import mindustry.type.*;
+import mindustry.world.Block;
+import mindustry.world.meta.BlockGroup;
```

Comment:
use wildcard imports:

```suggestion
import mindustry.world.*;
import mindustry.world.meta.*;
```